### PR TITLE
Upgrade tools to use redis v9

### PR DESCRIPTION
**Pull request by abezzub**
Upgrade `tools` to use Redis v9. This should resolve https://github.com/hibiken/asynq/issues/686

### CODE REVIEW

File: tools/asynq/cmd/root.go [hunk 1]
```diff
@@ -404,18 +404,22 @@ func getTLSConfig() *tls.Config {
...
-//         fmt.Fprintf(w, tmpl, u.Name, u.Addr, u.Age)
-//     }
-// }
+//
+//	printRows := func(w io.Writer, tmpl string) {
+//	    for _, u := range data {
+//	        fmt.Fprintf(w, tmpl, u.Name, u.Addr, u.Age)
+//	    }
+//	}
+//
```
Comment (hibiken) [inline tools/asynq/cmd/root.go:422]: Could you revert this change, I don't think it's related.

**Review by hibiken — CHANGES_REQUESTED** [review]
Thanks for opening a PR. Please address the open comment, then I'll approve this PR.

File: tools/asynq/cmd/root.go (hunk 1 above)
Comment (abezzub) [reply tools/asynq/cmd/root.go:422]: Sure, I assumed it is default go tool formatting

**Comment by abezzub** [comment]
@hibiken I updated the PR

File: tools/asynq/cmd/root.go (hunk 1 above)
Comment (hibiken) [reply tools/asynq/cmd/root.go]: we can address this in a separate PR :)
